iotdb-core/antlr/src/main/antlr4/org/apache/iotdb/db/qp/sql/IoTDBSqlParser.g4

```diff
@@ -210,7 +210,11 @@ countNodes
 
 // ---- Timeseries Where Clause
 devicesWhereClause
-    : WHERE deviceContainsExpression
+    : WHERE (deviceContainsExpression | templateEqualExpression)
+    ;
+
+templateEqualExpression
+    : TEMPLATE (operator_eq | OPERATOR_NEQ)  templateName=identifier
```

Comment:
Why these two operator, "operator_eq" and "OPERATOR_NEQ", is different in character case? Make the code style consistent.

Reply:
Fixed.